### PR TITLE
Change flip test

### LINKED ISSUE #30: Implementing `flip` function and writing 3-5 unit tests

### PR DESCRIPTION
**Pull request by Marcony1**
Hi guys, for some reason the tests are not running on my computer, could you take a look, please?

### CODE REVIEW

File: tests/test_flip.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    img = load_image('tests/img/raw/meme.jpg')
+    flipped_img = flip(img, v=1)
+
+    assert flipped_img.get_dimensions() == img.get_dimensions()
+
+    # Check if the top row of pixels is the same in both images (since it's flipped vertically)
+    assert np.array_equal(flipped_img.image[0, :], img.image[-1, :])
+
+    # Save the flipped image to the results directory
+    flipped_img.save(os.path.join(RESULTS_DIR, 'flipped_vertically.jpg'))
```
Comment (SoloSynth1) [inline tests/test_flip.py]: Will the saved image be tested afterwards?

**Review by SoloSynth1 — COMMENTED** [review]
@Marcony1 Please see comment. Also, I feel like we should make the unit test function names more descriptive, as it is one of the requirements listed out for [milestone 2](https://pages.github.ubc.ca/MDS-2023-24/DSCI_524_collab-sw-dev_students/release/milestone2/milestone2.html#write-test-cases-and-code-iteratively):
> ... Do not forget to also document your test functions. Check whether the code passes the unit tests you have written using the [pytest package](https://docs.pytest.org/en/latest/getting-started.html) for Python packages and ...

Please let me know what you think. Thanks!

**Comment by Marcony1** [comment]
> Hi guys, for some reason the tests are not running on my computer, could you take a look, please?

Fixed, thank you!

File: tests/test_flip.py (hunk 1 above)
Comment (Marcony1) [reply tests/test_flip.py]: No, in the end I think I was exporting them for me more than to test them. Removed in later update.

Comment (SoloSynth1) [reply tests/test_flip.py]: Yeah just saw that in the latest commit, I agree storing the flipped image and use them to compare the function output would be a better approach

**Review by SoloSynth1 — APPROVED** [review]
LGTM! Thanks @Marcony1

**Comment by SoloSynth1** [comment]
Merging to `main`.

**Pull request merged**